core/nodejsAction/Dockerfile

```diff
@@ -2,56 +2,20 @@ FROM buildpack-deps:trusty-curl
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Initial update and some basics.
-# This odd double update seems necessary to get curl to download without 404 errors.
-RUN curl -sL https://deb.nodesource.com/setup_0.12 | bash - && \
-  apt-get install -y nodejs imagemagick
-
```

Comment:
The gain is, you can use `npm` in a "normal" way. I do understand what the reasoning behind that was. To be honest, the new way I installed node didn't want to work with the link.